src/Features/LanguageServer/Protocol/Handler/Completion/CompletionResolveHandler.cs

```diff
@@ -50,35 +64,17 @@ public async Task<LSP.CompletionItem> HandleRequestAsync(LSP.CompletionItem comp
             }
 
             var completionService = document.Project.LanguageServices.GetRequiredService<CompletionService>();
-            var data = GetCompletionResolveData(completionItem);
-
-            CompletionList? list = null;
-
-            // See if we have a cache of the completion list we need
-            if (data.ResultId.HasValue)
+            var cacheEntry = GetCompletionListCacheEntry(completionItem);
+            if (cacheEntry == null)
             {
-                list = await _completionListCache.GetCachedCompletionListAsync(data.ResultId.Value, cancellationToken).ConfigureAwait(false);
+                // Don't have a cache associated with this completion item, cannot resolve.
+                return completionItem;
```

Comment:
Even though we log telemetry already, we should also log a loghub warning here.  If we get a feedback about a completion error it's really easy to tell exactly which completion request hit this (which we couldn't do by just looking at the telemetry).